### PR TITLE
Add location services

### LINKED ISSUE #99: Center the map on the user's location
Center the Leaflet map on the user's location if provided. The zoom should increase if the user's location is provided. If the location is not provided or geolocation services fails, set the center of the map to Cathy's coordinates.

### PR DESCRIPTION
**Pull request by nicoledaniels**
Implemented geolocation for the map. If the user provides their location, the map will center itself on their longitude and latitude. To make pins near the user easier to see, the map will also zoom in to 17 instead of 15. If the user does not have geolocation services or declines to give their location, set the center of the map to Cathy and keep the zoom at 15. Closes #99 

Also, added my name to the contributors list

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -4,13 +4,65 @@
...
+
+    // Options for geolocation services
+    var locationOptions = {
+      enableHighAccuracy: false,
+      timeout: 5000,
+      maximumAge: 60000
+    };
+
+    // Initialize the map
+    function loadMap(centerCoordinates, centerZoom){
```
Comment (vonbearshark) [inline main.js:41]: Nice succinct function for loading everything--great idea

**Review by vonbearshark — APPROVED** [review]
Looks great. I haven't had a chance to test the location services within the bounding box, but it puts me at Cathy when I'm in my apt well outside the box, which is exactly what we should do

**Comment by vonbearshark** [comment]
There seems to be this weird bug on Firefox that if a user selects "Not Now", the error function is never called: https://bugzilla.mozilla.org/show_bug.cgi?id=675533

**Comment by vonbearshark** [comment]
Likewise, Chrome only supports geolocation through https, which we don't have on our custom domain (which is also a GH Pages site) right now: https://developers.google.com/web/updates/2016/04/geolocation-on-secure-contexts-only. We'll need to set that up first.

**Comment by vonbearshark** [comment]
I reverted this PR, but you might be able to actually open it back up @nicoledaniels, because I can't seem to.

**Comment by nicoledaniels** [comment]
Don't think you can re-open merged and then un-merged PRs http://stackoverflow.com/questions/12674304/github-reopening-a-merged-pull-request

**Comment by vonbearshark** [comment]
_shrug_. I reopened #99

**Pull request merged**